app/lib/actions/katello/host/update_content_overrides.rb

```diff
@@ -2,25 +2,17 @@ module Actions
   module Katello
     module Host
       class UpdateContentOverrides < Actions::EntryAction
-        def plan(host, content_override_params, prune_invalid_content_overrides = true)
```

Comment:
Curious what `prune_invalid_content_overrides` was and why it was removed

Reply:
If it's really dead I've no problem with removing it!

Reply:
Both are acceptable I guess. Updated anyway